python/pyrogue/pydm/widgets/command_tree.py

```diff
@@ -139,6 +154,12 @@ def __init__(self,*,path,top,parent,command):
 
             self._top._tree.setItemWidget(self,3,self._widget)
 
+            width = fm.width('0xAAAAAAAA    ')
```

Comment:
`0xAAAAAAAA    ` seems strange.  Do you need all that white-space in the python string argument?